dist/ember-charts.js

```diff
@@ -5304,27 +5304,50 @@ define('ember-charts/mixins/pie-legend', ['exports', 'module', 'ember'], functio
       // cause labelTop to exceed the space allocated for the legend.
       var labelTop = 0;
 
-      var arr = labelStrings.slice(1).length;
-      for (var i = 0, len = arr.length; i < len; i++) {
-        nextLabel = arr[i];
-        currentText = row.text();
-        row.text("" + currentText + ", " + nextLabel);
-        rowNode = row.node();
-        if (rowNode.getBBox().width > this.get('legendWidth')) {
-          if (labelTop + rowNode.getBBox().height > this.get('maxLabelHeight')) {
-            row.text("" + currentText + ", ...");
-            break;
-          } else {
-            row.text("" + currentText + ",");
-            labelTop += rowNode.getBBox().height;
-            row = labels.append('text').text(nextLabel).attr(this.get('legendLabelAttrs')).attr('dy', labelTop);
+      var remainingLabelStrings = labelStrings.slice(1);
+      var _iteratorNormalCompletion = true;
+      var _didIteratorError = false;
+      var _iteratorError = undefined;
+
+      try {
+        for (var _iterator = remainingLabelStrings[Symbol.iterator](), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
```

Comment:
`Symbol` causes tests to fail on Travis CI, probably because it's not supported on all browsers. Super weird that Babel it this way. Why not just do the standard translation to a `for` loop? 

In any case, a `forEach` loop also doesn't work, because this loop contains a `break`, so I'm just gonna switch back to using a standard `for` loop